cms/envs/devstack.py

```diff
@@ -128,6 +128,7 @@ def should_show_debug_toolbar(request):
 ################################ SEARCH INDEX ################################
 FEATURES['ENABLE_COURSEWARE_INDEX'] = False
 FEATURES['ENABLE_LIBRARY_INDEX'] = False
+FEATURES['ENABLE_CONTENT_LIBRARY_INDEX'] = True
```

Comment:
Note to self: We will want to mark Elasticsearch as required by Studio in devstack's docker-compose.yml file.